commands/operator-sdk/cmd/up/local.go

```diff
@@ -165,21 +165,24 @@ func upLocalAnsible() {
 	printVersion()
 	log.Infof("watching namespace: %s", namespace)
 	done := make(chan error)
+	cMap := proxy.NewControllerMap()
 
 	// start the proxy
 	err = proxy.Run(done, proxy.Options{
-		Address:    "localhost",
-		Port:       8888,
-		KubeConfig: mgr.GetConfig(),
-		Cache:      mgr.GetCache(),
-		RESTMapper: mgr.GetRESTMapper(),
+		Address:       "localhost",
+		Port:          8888,
+		KubeConfig:    mgr.GetConfig(),
+		Cache:         mgr.GetCache(),
+		RESTMapper:    mgr.GetRESTMapper(),
+		Manager:       mgr,
```

Comment:
Now that we're passing the manager itself, it seems a little silly to pass all these other bits of data we get from it. Does it make sense for the proxy to just get those itself now? Not a blocker, but something to think about. If you want to go ahead and merge as-is, perhaps just file a github issue to consider it if you think it's a legit idea.

Reply:
Alternatively, we make it a runnable and have the manager mange this using the Dependency Injection from that?

Reply:
So actually we do not need to pass the manager into the proxy any more. Initially I was creating controllers from within the proxy and Shawn showed me how that was the wrong approach. I am removing this being passed in. I don't disagree that it might make sense having the proxy get those values but I will not be modifying the default behavior here.